here-naksha-app-service/src/test/java/com/here/naksha/app/service/CreateFeatureTest.java

```diff
@@ -232,10 +224,11 @@ void tc0305_testCreateFeaturesWithDupIds() throws Exception {
     assertEquals(200, response.statusCode(), "ResCode mismatch");
 
     // When: Create Features request is submitted with the same Ids
-    final String expectedBodyPart = loadFileOrFail("CreateFeatures/TC0305_createFeaturesWithDupIds/feature_response_part.json");
-    response = getNakshaClient().post("hub/spaces/" + SPACE_ID + "/features", bodyJson, streamId);
+    final String request = loadFileOrFail("CreateFeatures/TC0305_createFeaturesWithDupIds/request.json");
+    response = getNakshaClient().post("hub/spaces/" + SPACE_ID + "/features", request, streamId);
 
     // Then: Perform assertions
+    final String expectedBodyPart = loadFileOrFail("CreateFeatures/TC0305_createFeaturesWithDupIds/feature_response_part.json");
```

Comment:
we also need custom assertion to validate response contains `inserted` array of Ids matching with IDs available in `features` array in the response.